run_ansible.sh

```diff
@@ -115,9 +115,29 @@ pipenv_init() {
       rm /tmp/get-pipenv.py
     fi
 
-    wget https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py --directory /tmp
-    sudo python /tmp/get-pipenv.py
-    exit
+    case "$(uname -s)" in
+      Darwin)
+        echo 'Mac OS X'
+        brew install pipenv
+        ;;
+
+      Linux)
+        echo 'Linux'
+        if [ "$(which apt)" != "" ]; then
+          apt update
+          apt install -y python3-pip
+          pip3 install pipenv || exit 1
```

Comment:
this might be a good time to recommend not installing system wide?  

So, 
```pip3 install pipenv --user
```

However, that involved also insuring that $HOME/.local/bin is in the path (from .bashrc for example).